src/tests/Microsoft.PowerFx.Interpreter.Tests/ConfigTests.cs

```diff
@@ -217,6 +218,53 @@ public async Task Shared()
             Assert.Equal("28", result4.ToObject());
         }
 
+        [Fact]
+        public void RecalcEngine_Symbol_CultureInfo()
```

Comment:
>RecalcEngine_Symbol_CultureInfo [](http://example.com/codeflow?start=20&length=31)

// Parse in locale1, execute in locale2, on a machine with default locale3 (ignored)
Assert.Equal("212,345/1", 
 (engine.EvalAsync("Text(212345,1)", CancellationToken.None, 
 options: fr_ParserOptions, 
 runtimeConfig: fa-IR
 ).Result.ToObject());